src/main.c

```diff
@@ -165,14 +165,14 @@ void promptHandler(char *word) {
 }
 
 void resetBank() {
-    for (short i = 0; i < 26; i++) {
-        letterBank[i] = alphabet[i];
+    for (char i = 0; i < 26; i++) {
+        letterBank[i] = 'A' + i;
```

Comment:
No need to write out all the letters; `char` is just a number, and letters are also just a number. You can add directly to them to get the next letter; in this case, all uppercase alphabet letters are consecutive starting from `'A'` in ASCII.